mods/ra/maps/sarin-gas-3-controlled-burn/controlledburn.lua

```diff
@@ -99,6 +107,30 @@ PowerDownTeslas = function()
 	end
 end
 
+PrepareFinishingHunt = function(player)
+	local buildings = GetBaseBuildings(player)
+
+	Trigger.OnAllKilledOrCaptured(buildings, function()
+		Utils.Do(player.GetGroundAttackers(), function(actor)
+			actor.Hunt()
```

Comment:
Why is this not using `IdleHunt` this means that the actors will stop as soon as they destroyed one target.

Reply:
Because I requested it. All units should be attacking the player, not just idling ones. For instant there is a mammoth tank patrolling, you would want it to start hunting instead of continuing its patrol

Reply:
Hunt loops from what I've seen. Even once every target on the map is gone, hunters wait for new targets to attack. I have favored IdleHunt before partly 'cause it's possible (though rare) to get Nudged out of the activity and left idle.

Hunt is also a queued activity, so I will add a Stop to get patrolling USSR units to hunt right away, rather than when they reach the last patrol location.

Reply:
fwiw it should only be possible to nudge idle actors, meaning it's quite unlikely that hunt is stopped by a nudge

Reply:
> Because I requested it. All units should be attacking the player, not just idling ones. For instant there is a mammoth tank patrolling, you would want it to start hunting instead of continuing its patrol

You misunderstand what IdleHunt is doing. It is not using idling unit but makes unit hunt when they go into the idling state again. This needs to be changed back.